docs/ensuring-repro/openscpca-nf/index.md

```diff
@@ -0,0 +1,23 @@
+# The OpenScPCA Nextflow workflow
+
+In addition to the `OpenScPCA-analysis` repository, the Data Lab maintains [a separate repository called `OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+The `OpenScPCA-nf` repository holds a [Nextflow workflow](https://www.nextflow.io/) which we use to generate results from mature `OpenScPCA-analysis` modules.
+
+Specifically, once an analysis module has been completed, the Data Lab will port the analysis code into the `OpenScPCA-nf` repository and add it to the Nextflow workflow.
```

Comment:
I felt like this wasn't really making the previous sentence more specific, but rather adding context, so I removed the first word...
```suggestion
As analysis modules are completed, the Data Lab will port the analysis code into the `OpenScPCA-nf` repository and add it to the Nextflow workflow.
```